RoddyCore/src/de/dkfz/roddy/knowledge/files/BaseFile.groovy

```diff
@@ -685,11 +686,15 @@ abstract class BaseFile<FS extends FileStageSettings> extends FileObject {
             }.flatten().join("\n\t")
             sb << "\nNote these may not have matched because they may be specific for a tool!"
 
-            throw new ConfigurationError(sb.toString(), baseFile.executionContext.configuration)
+            if (Roddy.getFeatureToggleValue(AvailableFeatureToggles.FailOnAutoFilenames))
+                throw new ConfigurationError(sb.toString(), baseFile.executionContext.configuration)
+            else
+                logger.sometimes(sb.toString())
         } else {
             // Check whether the path exists and create it if necessary.
             if (context.getExecutionContextLevel().allowedToSubmitJobs && !FileSystemAccessProvider.getInstance().checkDirectory(patternResult.x.getParentFile(), context, true)) {
-                throw new IOException("Output path could not be created for file: " + baseFile)
+                if (Roddy.getFeatureToggleValue(AvailableFeatureToggles.FailOnAutoFilenames))
+                    throw new IOException("Output path could not be created for file: " + baseFile)
```

Comment:
The message seems not to fit to the toggle. Should that not be "Refuse to create automatically named file ${baseFile.toString()} (${AvailableFeatureToggles.FailOnAutoFilenames} toggle is set to 'true')", or so?